src/v2/view-builder/views/signin/SignInWithIdps.js

```diff
@@ -4,10 +4,13 @@ import hbs from 'handlebars-inline-precompile';
 export default View.extend({
   className: 'sign-in-with-idp',
   template: hbs`
-    {{#if addSeparateLine}}
+    {{#if addSeparateLineAtTop}}
```

Comment:
since we got a chance to refactor.. can you please rename this to `addDividerTop` and `addDividerBottom` 
`class="separation-line"` to `class="form-divider"`

Reply:
I can update the arguments to `addDividerTop` and `addDividerBottom`, however updating the class is a bit more risky since it's referenced in other places: in RedirectIdPView.js and of course the _identify.css file. Also I noticed that `CustomButtons.js` and `SignInWithDeviceOption.js` follow the same pattern in using the `seperation-line` class. I don't think it makes sense to just update it for `SignInWithIdps.js` and not others. Lastly, `form-divider` is already a special class that has some css attached to it in _forms.scss, I don't think we want this css applied in our case here hence why we may have chosen a different class to begin with.